packages/core/src/components/Nav/navItem.scss

```diff
@@ -72,58 +73,62 @@
     text-decoration: none;
     color: inherit;
   }
-}
-
-.cds-navItemSvg {
-  color: inherit;
-}
 
-.cds-navItemExpander svg {
-  transition: transform 0.3s ease-in-out;
-}
+  &-icon {
```

Comment:
It's hard to tell, but this class is the only thing I added to this file.

Reply:
Thanks for mentioning this, the diff was making my head spin initially!